### PR TITLE
Don't fix prawn version unless you're using Ruby 1.9

### PR DESCRIPTION
**Pull request by greysteil**
That. The JRuby failure is a little annoying, but should be fixed when 9000 is released in the next couple of months.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,17 @@
 language: ruby
+
+sudo: false
```
Comment (greysteil) [inline .travis.yml]: This just opt into using Travis's new infrastructure btw, which is faster. There's no way you need sudo  access to run your test suite, so no downside.

**Comment by greysteil** [comment]
@mgartner - are you happy with this? Would love to get it merged so we can bump the gem and get the security patch for `rest-client`

**Comment by mgartner** [comment]
I'd prefer that it didn't break JRuby. Do you know why that build is failing?

**Comment by greysteil** [comment]
Currently Travis's `jruby` rvm points at `jruby_19mode`, and Prawn 2.0 doesn't support Ruby 1.9.3. Once JRuby 9000 is out it'll switch to point at that, which will fix travis. Alternatively, we can set
````
env:
  JRUBY_OPTS=--2.0
```
which is what Prawn themselves do.

**Comment by mgartner** [comment]
:+1: thanks for the change - I'll merge once the build passes.

**Comment by greysteil** [comment]
Cool - give me a moment, just playing with the build matrix!

**Comment by greysteil** [comment]
@mgartner - all yours!

**Comment by greysteil** [comment]
@mgartner - apologies, my finger slipped!

Looks like the failure now is unrelated to this change - it's a 422 from you guys.

**Comment by mgartner** [comment]
@greysteil just push this in 1.9.1. Thanks for your contributions!

**Comment by greysteil** [comment]
Awesome, thanks @mgartner!

**Pull request merged**